var/spack/repos/builtin/packages/nekrs/package.py

```diff
@@ -111,26 +135,14 @@ def setup_build_environment(self, env):
         env.set("OCCA_VERBOSE", "1")
         self._setup_runtime_flags(env)
 
-    def setup_run_environment(self, env):
-        # The 'env' is included in the Spack generated module files.
-        self._setup_runtime_flags(env)
-
     def setup_dependent_build_environment(self, env, dependent_spec):
         # Export OCCA_* variables for everyone using this package from within
         # Spack.
         self._setup_runtime_flags(env)
 
-    def install(self, spec, prefix):
-        script_dir = "scripts"
-
-        with working_dir(script_dir):
-            # Make sure nekmpi wrapper uses srun when we know OpenMPI
-            # is not built with mpiexec
-            if "^openmpi~legacylaunchers" in spec:
-                filter_file(r"mpirun -np", "srun -n", "nrsmpi")
-                filter_file(r"mpirun -np", "srun -n", "nrspre")
-                filter_file(r"mpirun -np", "srun -n", "nrsbmpi")
 
+class GenericBuilder(spack.build_systems.generic.GenericBuilder):
```

Comment:
Missing:
```python
import spack.build_systems
```
?

Reply:
I had, several times. Confused as to why that works for me. Lemme see if I can reproduce after cleaning.

Reply:
Yeah, it still works. Weird. I'll add explicit imports.